### PR TITLE
docs: unify with upstream readme

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Copy/paste/adjust some of the good parts from our upstream `readme.md` file
- Setup Material for MkDocs so we can use buttons
  - Add button that links to our GitHub app

## Context:

First PR to unify our homepage with the upstream `renovatebot/renovate` readme.
I've grabbed some good parts from the `readme.md` and put them on our homepage.

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -1,15 +1,49 @@
...
-## Renovate's purpose
+- Get automated Pull Requests to update your dependencies
+- Reduce noise by running Renovate on a schedule
+- Relevant package files are discovered automatically
+- Supports monorepo architectures like Lerna or Yarn workspaces with no extra configuration
+- Bot behavior is customizable via configuration files (config as code)
+- Use ESLint-like shared config presets for ease of use and simplifying configuration (JSON format only)
+- Lock files are supported and updated in the same commit, including immediately resolving conflicts whenever PRs are merged
+- Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement (npm packages only)
+- Open source (installable via npm/Yarn or Docker Hub) so can be self-hosted or used via GitHub App
```
Comment (HonkingGoose) [inline src/index.md:18]: We have way too many bullet points on the homepage like this, so we should figure out what we want to keep and what we can drop from this list.

File: src/index.md [hunk 2]
```diff
@@ -1,15 +1,49 @@
 ![Renovate banner](https://app.renovatebot.com/images/whitesource_renovate_660_220.jpg)
 
-# Renovate documentation
+# Renovate
 
-Renovate is an Open Source tool to automate:
+Automated dependency updates.
+Multi-platform and multi-language.
```
Comment (HonkingGoose) [inline src/index.md:6]: I like the "tagline" from the readme, so I've inserted it here as well. 😄

File: src/index.md [hunk 3]
```diff
@@ -1,15 +1,49 @@
...
+- Use ESLint-like shared config presets for ease of use and simplifying configuration (JSON format only)
+- Lock files are supported and updated in the same commit, including immediately resolving conflicts whenever PRs are merged
+- Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement (npm packages only)
+- Open source (installable via npm/Yarn or Docker Hub) so can be self-hosted or used via GitHub App
+
+## Supported Platforms
+
+Renovate works on the following platforms:
+
+- GitHub (.com and Enterprise)
```
Comment (HonkingGoose) [inline src/index.md:24]:
```suggestion
- GitHub (.com and Enterprise Server)
```

File: src/index.md [hunk 4]
```diff
@@ -30,3 +63,5 @@ You can run Renovate as:
 - an [Open Source npm package](https://www.npmjs.com/package/renovate)
 - a [pre-built Open Source image on Docker Hub](https://hub.docker.com/repository/docker/renovate/renovate)
 - a [free GitHub App](https://github.com/marketplace/renovate) that is hosted by WhiteSource
+
+[Install our GitHub app now](https://github.com/marketplace/renovate){ .md-button .md-button--primary }
```
Comment (HonkingGoose) [inline src/index.md:67]: How about using a button to nudge the reader to install the Renovate app on GitHub? For example:

install-github-app-button

**Review by HonkingGoose — COMMENTED** [review]
Let's see how you like buttons. 😉

File: src/index.md [hunk 5]
```diff
@@ -2,14 +2,47 @@
...
-- Showing the release notes
+## Why use Renovate?
 
-## Renovate's purpose
+- Get pull requests to update your dependencies and lock files
+- Reduce noise by scheduling when Renovate runs
+- Renovate finds relevant package files automatically
+- Renovate works with monorepos like Lerna or Yarn workspaces with no extra configuration
+- You can customize the bot's behavior with configuration files
+- Share your configuration with ESLint-like config presets (JSON format only)
```
Comment (rarkins) [inline src/index.md:15]:
```suggestion
- Share your configuration with ESLint-like config presets
```

File: src/index.md [hunk 6]
```diff
@@ -2,14 +2,47 @@
...
 
-- Detecting dependencies in a repository (Open Source and private/closed source)
-- Checking if there are dependency updates
-- Creating commits and Merge/Pull Requests to update dependencies
-- Showing the release notes
+## Why use Renovate?
 
-## Renovate's purpose
+- Get pull requests to update your dependencies and lock files
+- Reduce noise by scheduling when Renovate runs
```
Comment (rarkins) [inline src/index.md:11]:
```suggestion
- Reduce noise by scheduling when Renovate creates PRs
```

File: src/index.md [hunk 7]
```diff
@@ -2,14 +2,47 @@
...
+- Renovate finds relevant package files automatically
+- Renovate works with monorepos like Lerna or Yarn workspaces with no extra configuration
```
Comment (rarkins) [inline src/index.md:13]:
```suggestion
- Renovate finds relevant package files automatically, including in monorepos
```

**Review by rarkins — CHANGES_REQUESTED** [review]

File: mkdocs.yml [hunk 8]
```diff
@@ -109,6 +109,11 @@ markdown_extensions:
   # https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#admonition
   - admonition
 
+  # The Attribute Lists extension allows to add HTML attributes and CSS classes to almost every Markdown inline- and block-level element with a special syntax.
```
Comment (HonkingGoose) [inline mkdocs.yml:112]:
```suggestion
  # The Attribute Lists extension allows us to add HTML attributes and CSS classes to almost every Markdown inline- and block-level element with a special syntax.
```

File: src/index.md [hunk 9]
```diff
@@ -2,14 +2,46 @@
...
+## The Renovate Approach
+
+We believe everyone benefits from automation, whether it's a little or a lot.
+This means that Renovate:
+
+- Adapts to your workflow
+- Allows you to configure its behavior
+- Will autodetect settings where possible
+
+## Renovate's Purpose
```
Comment (HonkingGoose) [inline src/index.md:44]: Do we really _need_ the `The Renovate Approach` and `Renovate's Purpose` sections? I think the homepage would look better if we removed these sections, and let the main `Why use Renovate?` section entice the reader.

Comment (rarkins) [reply src/index.md:44]: Yes, we can remove

**Review by rarkins — APPROVED** [review]

**Pull request merged**